requirements-dev.txt

```diff
@@ -1,2 +1,3 @@
-pytest==2.7.2
+pytest==2.9.2
 pytest-cov==2.1.0
+git+git://github.com/tilery/utilery.git
```

Comment:
Happy to! I'll try to see how else I can help. I have a lot to learn in
testing, this should be fun. You've been warned!  

  
On Aug 10 2016, at 3:27 pm, shongololo <notifications@github.com> wrote:  

> In [requirements-dev.txt](https://github.com/shongololo/aiovectortiler/pull/
3#discussion_r74323450):

>  
>  
>     >  pytest-cov==2.1.0

>     > +git+git://github.com/tilery/utilery.git

>  
>

> Starting a tests discussion in issues and I'm also going to create a tests
branch. Thanks for getting the ball rolling.

>

> --  
You are receiving this because you authored the thread.  
Reply to this email directly, [view it on GitHub](https://github.com/shongolol
o/aiovectortiler/pull/3/files/aaa13ca1cc41472867ad2c01ade11d14563926f3#r743234
50), or [mute the thread](https://github.com/notifications/unsubscribe-auth/AA
gcFJAQ0898saGe_wh6A7W9C_WfO9wEks5qejRBgaJpZM4JhYFH).![](https://github.com/not
ifications/beacon/AAgcFPMfOljLJ8Vq38gF9xNkqgWl4Epjks5qejRBgaJpZM4JhYFH.gif)